src/main/java/com/microsoft/sqlserver/jdbc/SQLServerBulkCopy.java

```diff
@@ -1136,7 +1136,10 @@ private void writeTypeInfo(TDSWriter tdsWriter,
                     tdsWriter.writeByte((byte) srcScale);
                 }
                 break;
-
+            case microsoft.sql.Types.SQL_VARIANT: 
+                tdsWriter.writeByte(TDSType.SQL_VARIANT.byteValue());
+                tdsWriter.writeInt(8009); //write lenght of sql variant 8009
```

Comment:
this is the second time I see this number. if it is a constant documented for sql variant, let's replace the magic number with a final static constant variable